components/Ink/src/MDCInkView.m

```diff
@@ -153,16 +153,6 @@ - (void)startTouchBeganAnimationAtPoint:(CGPoint)point
     inkLayer.inkColor = self.inkColor;
     inkLayer.maxRippleRadius = self.maxRippleRadius;
     inkLayer.animationDelegate = self;
-
-    switch (self.inkStyle) {
```

Comment:
Are we intentionally deleting this? It looks like a accidental deletion given the description of the PR.

Reply:
Yes, it's an intentional deletion. inkStyle shouldn't be used with the new version of the ripple since there isn't a distinction between bounded an unbounded ripples.